src/components/pages/home/applyCommunityVerify/verifyPhoneAndEmail.js

```diff
@@ -4,6 +4,8 @@ import PhoneInput, { isPossiblePhoneNumber } from "react-phone-number-input";
 import { toast } from "react-toastify";
 import OtpInput from "react-otp-input";
 import { checkUniquePhone } from "../../../../utils/uniqueUser";
+import { log_event } from "../../../../utils/utilityFunctions";
+import { api_link } from "../../../../utils/supabase";
```

Comment:
why do we have it in underscore naming, is there a predefined naming conventions that we are following?  can we follow camelCase

Reply:
The underscore naming conventions is followed for non react code and every other hook and component follows camelCase this keeps the code visually separated !